### ISSUE TITLE
Notation .. does not recognize printing boxes well enough

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#12385]
#### Description of the problem

```coq
Require Import Coq.Lists.List.
Notation "[ x ; y ; .. ; z ]" :=  (cons x (cons y .. (cons z nil) ..)) (format "'[hv ' [ '[' x ']' ;  '/' '[' y ']' ;  '/' .. ;  '/' '[' z ']' ] ']'") : list_scope.
(* Error:
The format is not the same on the right- and left-hand sides of the special token "..". *)
```
I think this is a bug, but maybe @herbelin will point out something I'm missing.

#### Coq Version

8.11.1

**Comment by herbelin** [comment]
Hi @JasonGross, this is addressed by #10835, which is in 8.12.

**Comment by JasonGross** [comment]
Neat, thanks!
